cfme/tests/control/test_bugs.py

```diff
@@ -26,6 +29,104 @@
 ]
 
 
+def create_policy_profile(request):
+    random_string = fauxfactory.gen_alpha()
+    policy = VMControlPolicy(random_string)
+    policy.create()
+    policy_profile = PolicyProfile(random_string, [policy])
+    policy_profile.create()
+
+    @request.addfinalizer
+    def _delete():
+        while policy_profile.exists:
+            policy_profile.delete()
+        if policy.exists:
+            policy.delete()
+
+    return policy_profile
+
+
+def create_policy(request):
+    policy = VMControlPolicy(fauxfactory.gen_alpha())
+    policy.create()
+
+    @request.addfinalizer
+    def _delete():
+        while policy.exists:
+            policy.delete()
+
+    return policy
+
+
+def create_condition(request):
+    condition = VMCondition(
+        fauxfactory.gen_alpha(),
+        "fill_field(VM and Instance : Boot Time, BEFORE, Today)"
+    )
+    condition.create()
+
+    @request.addfinalizer
+    def _delete():
+        while condition.exists:
+            condition.delete()
+
+    return condition
+
+
+def create_action(request):
+    action = Action(
+        fauxfactory.gen_alpha(),
+        action_type="Tag",
```

Comment:
Any future value from taking action_type/values as arguments with defaults? Could easily test different actions in the later? Not necessary, just a thought.